app/src/main/res/values/strings.xml

```diff
@@ -110,5 +110,8 @@
     <string name="chat">Chat</string>
     <string name="reviews">Reviews</string>
     <string name="log_out">Log out</string>
+    <string name="title_activity_rdetails_fragment">RdetailsFragment</string>
```

Comment:
The title should be a human readable string, like `Post details`

Reply:
done